allensdk/brain_observatory/ecephys/ecephys_project_cache.py

```diff
@@ -98,23 +100,34 @@ def _get_probes(self):
             probes["lfp_sampling_rate"] / probes["lfp_temporal_subsampling_factor"])
         return probes
 
-
     def _get_channels(self):
         path = self.get_cache_path(None, self.CHANNELS_KEY)
         return call_caching(self.fetch_api.get_channels, path, strategy='lazy', **csv_io)
 
-
     def _get_units(self, **kwargs):
         path = self.get_cache_path(None, self.UNITS_KEY)
         get_units = functools.partial(
-            self.fetch_api.get_units, 
-            amplitude_cutoff_maximum=None, # pull down all the units to csv and filter on the way out
-            presence_ratio_minimum=None, 
+            self.fetch_api.get_units,
+            amplitude_cutoff_maximum=None,  # pull down all the units to csv and filter on the way out
+            presence_ratio_minimum=None,
             isi_violations_maximum=None
         )
         units = call_caching(get_units, path, strategy='lazy', **csv_io)
-
-        units =units[
+        units = units.rename(columns={
+            'PT_ratio': 'waveform_PT_ratio',
+            'amplitude': 'waveform_amplitude',
+            'duration': 'waveform_duration',
+            'halfwidth': 'waveform_halfwidth',
+            'recovery_slope': 'waveform_recovery_slope',
+            'repolarization_slope': 'waveform_repolarization_slope',
+            'spread': 'waveform_spread',
+            'velocity_above': 'waveform_velocity_above',
+            'velocity_below': 'waveform_velocity_below',
+
```

Comment:
not functionally important, but ... random blank line?

Reply:
It's a leftover organizational strategy I was using for the different categories of renaming that were desired from the issue.

I've gone ahead and removed those blank lines.